cart-service/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM openjdk:11-jdk-slim
+WORKDIR /opt
+COPY target/*.jar /opt/app.jar
```

Comment:
Update the docker file according to 
https://github.com/NashTech-Labs/car-demo/blob/main/order-service/Dockerfile

Reply:
Done